doc/nrf/images/matter_architecture.svg

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<!-- Generated by Microsoft Visio, SVG Export matter_architecture.svg Page-1 -->
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:ev="http://www.w3.org/2001/xml-events"
+		width="6.0514in" height="3.39381in" viewBox="0 0 435.701 244.354" xml:space="preserve" color-interpolation-filters="sRGB"
+		class="st9">
+	<style type="text/css">
+	<![CDATA[
+		.st1 {fill:#768692;stroke:none;stroke-width:1}
+		.st2 {fill:#feffff;font-family:Calibri;font-size:0.833336em}
```

Comment:
I checked what fonts we typically use in SVGs in the documentation and there seems to be no consistency:
```
:~/src/ncs/nrf/doc$ find -name '*.svg' | grep -v _build | xargs grep -hoP 'font-family:\s*\w+' | sort | uniq -c | sort -nr
    332 font-family:Calibri
     56 font-family:Arial
     37 font-family:Verdana
     22 font-family:sans
      4 font-family: Helvetica
      2 font-family:Wingdings
      2 font-family:Gotham
      1 font-family:Consolas
```
However, when I tested some of those SVGs on Ubuntu it seems that Arial has the least rendering issues on Ubuntu. Calibri often results in text sticking out of a box it belongs to. I think it may be worth discussing that among tech writers at some point and agree on a common font. Maybe installing some windows fonts on Linux will help with Calibri issues - I can try that, too.

Another idea related to SVG files is that their size can be minimized using some SVG compressors available online, e.g. https://jakearchibald.github.io/svgomg/ (not sure if Visio or whatever tool you use supports exporting to minimized SVGs, without whitespaces etc).  Anyway, it's just a hint.

Reply:
This is because we didn't edit them all to Arial when we decided to use Arial, but we're doing it on the go when we update the page/the figures. Which reminds me I need to change it for this figure. Forgot you mentioned it at the standup.

Reply:
Ah, great.